github-action-example/workflow.yml

```diff
@@ -0,0 +1,17 @@
+name: "JFrog CLI Example"
+on: push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Setup JFrog CLI
+        uses: jfrog/setup-jfrog-cli@v1
+        env:
+          JF_ARTIFACTORY_SERVER: ${{ secrets.JF_ARTIFACTORY_SERVER_TOKEN }}
+      - name: Run JFrog CLI
+        run: |
+          jfrog rt bce # Collect environment variables for the build
+          jfrog rt bp # Publish build info
```

Comment:
In my opinion, placing the description comments above the lines they describe, make the source more readable.